methods/3.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "3",
+  "description": "",
+  "status": "",
+  "registry": "Ceramic Network",
+  "contact": "<a href=\"mailto:oed@3box.io\">Joel Thorstensson</a>",
```

Comment:
<script>alert(1);</script>

Reply:
prefer json to not include XSS vectors.

Reply:
> At present, we allow HTML markup in all entries (this is nothing new, we've operated like this for a while).

This PR is the first time we have added JSON files which contain html markup to the registry, and we should not start by encouraging dom based xss via respec.

is contact always an email? why not call it "email" and make it an email?

Reply:
> is contact always an email? why not call it "email" and make it an email?

No, it isn't. Sometimes it's an email. Sometimes it's a website. Sometimes it's just a name. Sometimes it's just the name of a company. So, what of those options are we going to force people to do (because they all seem like valid options, IMHO).

Reply:
ok thats fail, I think its easier to express that as optional fields.... Like this:

```yaml
title: DID Method Registry Entry
description: This schema defines the shape of did method registrations in the did spec registries
type: object
required:
  - status
  - name
  - specification
properties:
  status:
    type: string
    enum: ["accepted", "withdrawn", "deprecated"]
  name:
    type: string
    example: "did:example"
  description:
    type: string
    example: "did:example is reserved for examples."
  specification:
    type: string
    example: "https://workday.github.io/work-did-method-spec/"
  contactName:
    type: string
    example: "Example Corp. Support"
  contactEmail:
    type: string
    example: "support@example.com"
  contactWebsite:
    type: string
    example: "example.com"
  verifiableDataRegistry:
    type: string
    example: "Bitcoin / Ethereum + IPFS / Hyperledger"
  rubricEvaluation:
    type: string
    example: ""
additionalProperties: false
```

Reply:
`additionalProperties: false` <- this is key.

Reply:
I'm reworking the schema you have above into the conversion script (I don't think we need maxLength, but *shrug* -- if you want it go ahead and put it in there). Bonus points if you want to work the JSON Schema above into a Github merge test for all PRs in the methods/ directory.

Reply:
I implemented the schema above in the latest PR commits. The only change I didn't take was the "accepted" language... I chose to use "registered" instead.

We may not even want to have "registered", since it's obvious that it's registered if it's in the DID Spec Registries. So, it's either blank or "withdrawn"/"deprecated". We may even want to collapse both of those entries into "deprecated".